config_test.go

```diff
@@ -147,6 +147,69 @@ func TestConfigs_appPaths(t *testing.T) {
 	}
 }
 
+func TestConfigs_mappingPaths(t *testing.T) {
+	tests := []struct {
+		name        string
+		configs     Configs
+		wantErr     bool
+		createFiles []string
+	}{
+		{
+			name:    "no mapping file",
+			configs: Configs{},
+			wantErr: false,
+		},
+		{
+			name:        "single mapping file",
+			configs:     Configs{MappingFile: os.TempDir() + "TestConfigs_mappingPaths_mapping.txt"},
```

Comment:
NOTE: This is available since go 1.15: https://pkg.go.dev/testing#T.TempDir, a helper function which cleans up the temp dir automatically.